modules/data.land/R/extract_veg.R

```diff
@@ -1,27 +1,53 @@
 ##' Function queries a DB to extract veg info downstream
 ##' @name extract_veg
-##' @title load_veg
+##' @title extract_veg
+##' 
+##' @param new_site new_site object passed from ic_process includes lat, lon, id, and name
+##' @param start_date "YYYY-MM-DD"
+##' @param end_date "YYYY-MM-DD"
+##' @param source taken from input$source, passed from ic_process
+##' @param gridres only used for source = "FIA"
+##' @param format_name DEFAULT=NULL
+##' @param machine_host passed from ic_process
+##' @param dbparms taken from settings object, passed from ic_process
+##' @param outfolder passed from ic_process, location where to store files
+##' @param overwrite DEFAULT = FALSE
+##' @param ... Additional parameters
+##'
+##' @return results object to be passed back to get.veg.module
 ##' @export
 ##' @author Istem Fer
```

Comment:
general comment: @helge22a you should add yourself to the author list of functions you created or made major modifications to